index.js

```diff
@@ -1,26 +1,120 @@
 // 1.Fibonacci
 
-// function fibo(n) {
-//   return n <= 1 ? n : fibo(n - 1) + fibo(n - 2);
-// }
-// fibo(5);
-// fibo(6);
-// fibo(7);
+function fibo(n) {
+  return n <= 1 ? n : fibo(n - 1) + fibo(n - 2);
+}
+fibo(5);
+fibo(6);
+fibo(7);
 
 // 2. Implementing with reduce
 
 //concat;
+let arr = [1, 2];
+let arr1 = [3, 4];
+
+const concatFn = (...array) => {
```

Comment:
this function should not only using arrays. 
`
const arr = [1,2,3]
const newArr = arr.concat([4,5,6])
const newArr2 = arr.concat(4)
`